packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -108,7 +122,7 @@ export class ColorPicker extends React.Component<IColorPickerProps, any> {
     if (newColor.str !== this.state.color.str) {
       this.setState({
         color: newColor
-      });
+      } as IColorPickerState);
 
       if (onColorChanged) {
         onColorChanged(newColor.str);
```

Comment:
Agree but then we should avoid doing this in other components as well. I see we do this in sync in other components. Fixed it with additional test to validate callback handler.